src/services/legal-services.ts

```diff
@@ -29,12 +28,11 @@ export default class LegalServices {
 
   /**
    * Fetches the first or only filing.
-   * This is expected to be a draft IA or Registration.
+   * This is probably a draft Amalgamation, IA or Registration.
    * @param tempId the temp registration number
    * @returns a promise to return the draft filing, else exception
    */
-  // eslint-disable-next-line max-len
-  static async fetchFirstOrOnlyFiling (tempId: string): Promise<AmalgamationFilingIF | IncorporationFilingIF | RegistrationFilingIF> {
+  static async fetchFirstOrOnlyFiling (tempId: string): Promise<any> {
```

Comment:
When this function is used by amalgamation code, the type of filing could actually be any of our filings in the ledger.

Ditto for tasks below.